ocaml-eglot-util.el

```diff
@@ -99,5 +99,16 @@
       (list :start start
             :end (ocaml-eglot-util--position-increase-char start "_")))))
 
+(defun ocaml-eglot-util--visit-file (strategy current-file new-file range)
+  "Visits a referenced document, NEW-FILE at position  start of RANGE.
+The STRATEGY can be `'new' `'current' or `'smart'.  CURRENT-FILE is used
+as a smart strategy."
```

Comment:
```suggestion
  "Visits a referenced document, NEW-FILE at position  start of RANGE.
The STRATEGY can be `'new' `'current' or `'smart'.  The later opens a
new buffer if the destination is not in the CURRENT-FILE, ans uses the 
current buffer otherwise."
```